web/impact/impact/v1/views/base_list_view.py

```diff
@@ -67,10 +67,12 @@ def get(self, request):
         count, results = self.results(limit, offset)
         result = {
             "count": count,
-            "next": _next_url(base_url, limit, offset, count),
-            "previous": _previous_url(base_url, limit, offset, count),
             "results": results,
         }
+        if limit != "all":
```

Comment:
It seems weird that we are testing this condition in two widely separated parts of the same class. 
It also seems weird that we're going to a function to compute a result, and then modifying that result when we get back. Shouldn't we just compute and return this dict from the `results` function? (possibly calling a helper if that makes `results` go over ten lines)

EDIT: note that this change would bring this `get` function comfortably within our desired 10-line limit...